pom.xml

```diff
@@ -1,8 +1,8 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.axonivy.market</groupId><!-- your group id: e.g. com.axonivy.connector.<myconnector> or com.axonivy.utils.<myutil> -->
```

Comment:
groupId=com.axonivy.connector.threema  ... use it for all poms in this repo.
its easier to manage like this, as we have operate a maven mirror for these artifacts.

Reply:
changed groupId everywhere to match "com.axonivy.connector.threema"